Build configuration of es-dashboards

Trying to run `mvn install -Pkb-download`

gives the following error:

```
[ERROR] Failed to execute goal com.googlecode.maven-download-plugin:download-maven-plugin:1.4.2:wget (download-kibana) on project gn-es-dashboards: IO Error: Error while expanding C:\Users\B004114\Documents\git\geonetwork\core-geonetwork\es\es-dashboards\kibana-8.14.3-windows-x86.zip: archive is not a ZIP archive -> [Help 1]
```

The URL is configured as `https://artifacts.elastic.co/downloads/kibana/kibana-${es.version}-${kb.platform}.${kb.installer.extension}`. The configuration:

https://github.com/geonetwork/core-geonetwork/blob/86c8dc5eb5e1a272e13144fb644f80384d5c7797/pom.xml#L1601

https://github.com/geonetwork/core-geonetwork/blob/86c8dc5eb5e1a272e13144fb644f80384d5c7797/pom.xml#L1503-L1517

When checking https://www.elastic.co/downloads/past-releases/kibana-8-14-3, it seems that `kb.platform` should be set to `windows-x86_64` instead of `windows-x86`.

After fixing that, I tried to run `mvn exec:exec -Dkb-start`, but go the following error:

```
mvn exec:exec -Dkb-start
[INFO] Scanning for projects...
[INFO]
[INFO] -------------< org.geonetwork-opensource:gn-es-dashboards >-------------
[INFO] Building GeoNetwork dashboard app based on Kibana 4.4.10-SNAPSHOT
[INFO]   from pom.xml
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- exec:3.5.0:exec (default-cli) @ gn-es-dashboards ---
[ERROR] Command execution failed.
java.io.IOException: Cannot run program "kibana-8.14.3-windows-x86_64\bin\kibana" (in directory "C:\Users\B004114\Documents\git\geonetwork\core-geonetwork\es\es-dashboards"): CreateProcess error=2, The system cannot find the file specified
```

The executable configured on Windows should be `kibana-8.14.3\bin\kibana.bat` instead of `kibana-8.14.3-windows-x86_64\bin\kibana`.

**To Reproduce**
Follow the instructions on https://github.com/geonetwork/core-geonetwork/blob/86c8dc5eb5e1a272e13144fb644f80384d5c7797/es/es-dashboards/pom.xml

**Expected behavior**
I expect the build to succeed and Kibana to be downloaded, unpacked and run correctly.

**Screenshots**

<img width="701" height="39" alt="Image" src="https://github.com/user-attachments/assets/d0ed757b-5ffc-4c4d-b426-6004d487d543" />

**Log file**
N/A

**Desktop**
 - GeoNetwork Version 4.4.10-SNAPSHOT, commit 86c8dc5eb5.
 - OS: Windows

**Additional context**
N/A

--- a/es/README.md
+++ b/es/README.md
@@ -38,7 +38,7 @@ These configurations should not be used for a production deployment.
 3. Check that it is running using your browser:
    
    * Elasticsearch: http://localhost:9200
-   * Kibana: http://localhost:5601
+   * Kibana: http://localhost:5601/api/status
 
 ### Maven installation
 

--- a/es/es-dashboards/README.md
+++ b/es/es-dashboards/README.md
@@ -14,7 +14,7 @@
 3. Check that it is running using your browser:
    
    * Elasticsearch: http://localhost:9200
-   * Kabana: http://localhost:5601
+   * Kibana: http://localhost:5601/api/status
 
 ## Maven installation
 
@@ -24,14 +24,21 @@
    * initialize collection
    * start
 
-2. Use maven to download:
+2. Use Maven to download:
 
    ```
-   cd es/es-dashboard
+   cd es/es-dashboards
    mvn install -Pkb-download
    ```
 
-3. Run locally:
+3. Set Kibana base path in config/kibana.yml:
+
+   ```
+   server.basePath: "/geonetwork/dashboards"
+   server.rewriteBasePath: false
+   ```
+
+4. Run locally:
 
    ```
    mvn exec:exec -Dkb-start
@@ -41,7 +48,7 @@
 
 1. Download Kibana 8.14.3 from https://www.elastic.co/downloads/kibana
 
-2. Set Kibana base path and index name in config/kibana.yml:
+2. Set Kibana base path in config/kibana.yml:
 
    ```
    server.basePath: "/geonetwork/dashboards"
@@ -59,10 +66,10 @@
 
 ## Import Configuration
 
-1. Kibana should be running from:
+1. The Kibana status details should be available at:
 
    ```
-   http://localhost:5601
+   http://localhost:5601/api/status
    ```
 
 2. And should be visible within the geonetwork interface at:

--- a/es/es-dashboards/pom.xml
+++ b/es/es-dashboards/pom.xml
@@ -139,7 +139,7 @@
             <artifactId>exec-maven-plugin</artifactId>
             <inherited>false</inherited>
             <configuration>
-              <executable>kibana-${es.version}-${kb.platform}/bin/${kb.executable}</executable>
+              <executable>kibana-${es.version}/bin/${kb.executable}</executable>
             </configuration>
           </plugin>
         </plugins>
@@ -147,7 +147,6 @@
     </profile>
   </profiles>
   <properties>
-    <kb.executable>kibana</kb.executable>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
   </properties>
 </project>

--- a/pom.xml
+++ b/pom.xml
@@ -1511,7 +1511,7 @@
          <es.platform>windows-x86_64</es.platform>
          <es.installer.extension>zip</es.installer.extension>
          <kb.executable>kibana.bat</kb.executable>
-         <kb.platform>windows-x86</kb.platform>
+         <kb.platform>windows-x86_64</kb.platform>
          <kb.installer.extension>zip</kb.installer.extension>
        </properties>
      </profile>

